styles/style.css

```diff
@@ -6,3 +6,265 @@ Green: #00B172
 White: #FFF
 
 */
+/* 
+===================================================
+CLEARING BROWSER DEFAULTS AND SETTING GLOBAL STYLES
+===================================================
+ */
+body {
+  margin: 0;
+  font-size: 1.5rem;
+  font-weight: lighter;
+  color: #1a1a1a;
+}
+
+/* 
+==========================================
+LAYOUT
+==========================================
+ */
+
+section,
+header {
+  border: 1px red solid;
+}
+.row,
+col {
+  border: 1px green dashed;
+}
+/* layout guides, to be used in development only */
+nav {
+  width: 100%;
+  height: 4.5rem;
+  background-color: #fff;
+}
+.logo {
+  width: 10%;
+  margin: 1rem 0 0 1.5rem;
+}
+.hamburger {
+  display: none;
+  padding: 15px;
+  cursor: pointer;
+  #mobile_menu {
+    color: #00b172;
+  }
```

Comment:
This doesn't seem like it belongs here. Certainly not being used since these two elements don't interact.

Reply:
Noted